src/DssSpell.sol

```diff
@@ -59,187 +51,250 @@ contract DssSpellAction is DssAction {
+        // gap: 50 million USDS
+        // ttl: 24 hours
+        // tau: 7 days
+        // DDM Addresses:
+        // oracle: D3MOracle at 0x9dB0EB29c2819f9AE0A91A6E6f644C35a7493E9b
+        // plan: D3MOperatorPlan at 0xea2abB24bF40ac97746AFf6daCA0BBF885014b31
+        // pool: D3MAaveV3USDSNoSupplyCapTypePool at 0xbf674d0cD6841C1d7f9b8E809B967B3C5E867653
+        // aToken: 0x09AA30b182488f769a9824F15E6Ce58591Da4781
+        // operator: 0x298b375f24CeDb45e936D7e21d6Eb05e344adFb5
+        // stabledebt address: 0x779dB175167C60c2B2193Be6B8d8B3602435e89E
+        // variabledebt address: 0x2D9fe18b6c35FE439cC15D932cc5C943bf2d901E
+        // Additional Actions
+        // Expand DIRECT_MOM breaker to also include new DDM
+        // Note: this is already done within D3MInit.sol line 232
+
+        D3MInstance memory d3m = D3MInstance({
+            plan:   DIRECT_SPK_AAVE_LIDO_USDS_PLAN,
+            pool:   DIRECT_SPK_AAVE_LIDO_USDS_POOL,
+            oracle: DIRECT_SPK_AAVE_LIDO_USDS_ORACLE
+        });
+
+        D3MCommonConfig memory d3mCfg = D3MCommonConfig({
+            hub:         D3M_HUB,
+            mom:         D3M_MOM,
+            ilk:         "DIRECT-SPK-AAVE-LIDO-USDS",
+            existingIlk: false,
+            maxLine:     100 * MILLION * RAD,
+            gap:         50 * MILLION * RAD,
+            ttl:         24 hours,
+            tau:         7 days
+        });
+
+        D3MAaveUSDSPoolConfig memory aaveCfg = D3MAaveUSDSPoolConfig({
+            king:         MCD_PAUSE_PROXY,
+            ausds:        DIRECT_SPK_AAVE_LIDO_USDS_AUSDS,
+            usdsJoin:     USDS_JOIN,
+            usds:         USDS,
+            stableDebt:   DIRECT_SPK_AAVE_LIDO_USDS_STABLE_DEBT,
+            variableDebt: DIRECT_SPK_AAVE_LIDO_USDS_VARIABLE_DEBT
+        });
+
+        D3MOperatorPlanConfig memory operatorCfg = D3MOperatorPlanConfig({
+            operator: DIRECT_SPK_AAVE_LIDO_USDS_OPERATOR
+        });
+
+        D3MInit.initCommon({
+            dss:     dss,
+            d3m:     d3m,
+            cfg:     d3mCfg
+        });
+
+        D3MInit.initAaveUSDSPool({
+            dss:     dss,
+            d3m:     d3m,
+            cfg:     d3mCfg,
+            aaveCfg: aaveCfg
+        });
+
+        D3MInit.initOperatorPlan({
+            d3m: d3m,
+            operatorCfg: operatorCfg
+        });
+
+        // ---------- Update WBTC Legacy Vaults Parameters  ----------
+        // Forum: https://forum.makerdao.com/t/wbtc-changes-and-risk-mitigation-10-august-2024/24844/48
+        // Forum: https://forum.sky.money/t/wbtc-changes-and-risk-mitigation-10-august-2024/24844/52
+
+        // Decrease liquidation penalty for WBTC-A, WBTC-B, and WBTC-C from 13% to 0%
+        DssExecLib.setIlkLiquidationPenalty("WBTC-A", 0);
+        DssExecLib.setIlkLiquidationPenalty("WBTC-B", 0);
+        DssExecLib.setIlkLiquidationPenalty("WBTC-C", 0);
+
+        address spotter = DssExecLib.spotter();
```

Comment:
This line doesn't have any instruction comment. Also usually all declarations are done on top of the actions with public and immutable keywords.

Reply:
you are right, I had it like this because it was just grabbing a param for the function and it was like that in previous spells, I should have added a `Note:`, but i prefer your suggestion and the immutable addresses pattern, so I added it as an immutable address in the beginning. Check https://github.com/makerdao/spells-mainnet/pull/432/commits/e8a3716e08d47d2a9149d8070b8dcfbcbaef0a58